File: code_12.py
def find_sum(nums):
    total = 0
    for num in nums:
        total += num
    return total * 2  # Logical mistake: multiplying by 2 unnecessarily.

# Main code
numbers = [1, 2, 3, 4]
print("Sum of list is:", find_sum(numbers))  # Syntax mistake: missing colon in print.

# End message
print("Congratulations! You have qualified for the next round.")

File: cpp codes/code12.cpp
#include <iostream>
#include <vector>
using namespace std;

void team12() {
    vector<int> x = {1, 2, 3};
    cout << x.at(3) << endl; // Logical Error: Index out of range
    cout << "You qualify for the next round!" << endl;
}

int main() {
    team12();
    return 0;
}
